packages/math/graph/graph.pony

```diff
@@ -0,0 +1,27 @@
+use "collections/persistent"
+"""
+A Graph is defined as a Set vertices and a Set of Edges.
+In pure Graph Theory, the vertex set MUST NOT be empty
+but obviously that would be quite annoying, so we will
+allow it, and provide an IsValidGraph method which will
+return true IFF the vertex set is not the empty set.
+"""
+class Graph
+  var _vertices: Lists[Vertex]
+  var _edges: Lists[Edge]
+  
+  fun create() =>
+    _vertices = Lists[Vertex].empty()
+    _edges = Lists[Edge].empty()
+    
+  fun addVertex(vertex: Vertex): Bool ? =>
```

Comment:
The standard library uses "snake_case" style names for methods. For example, this should be `add_vertex`.

Reply:
You can check the official style guide [here](https://github.com/ponylang/ponyc/blob/master/STYLE_GUIDE.md).